_commands/tools/scp.md

```diff
@@ -3,22 +3,35 @@
 
 scp
 -------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`scp`, or _Secure Copy_, is used to securely transfer files based on the _Secure Shell protocol_ (`ssh`). Unlike `cp`, at least one of the source or destination should be a remote host.
 
-<!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ scp source user@host:destination
+$ scp user@host:source destination
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### 'ssh alias'
```

Comment:
backticks? also might be a good idea to put this in a section outside of "useful options / examples", maybe like "optional preliminary steps" idk